debian/compat

```diff
@@ -1 +0,0 @@
-9
```

Comment:
Why this file was removed?

Reply:
It showed warnings that compat 9 is deprecated thus I changed it to: debian/control:  debhelper-compat (=13). Additionally man 7 debhelper states: 

Note that debhelper does not provide debhelper-compat for experimental or beta compatibility levels; packages
experimenting with those compatibility levels should use debian/compat or DH_COMPAT.

So this file is not necessary, however if you like to have it back for experiments I'll re-add it with 13 contained.

Reply:
If it is in the debian-policy, I am fine with it. Thanks.